### DISCUSSION TITLE
Migrate replication tasks to the new harbor

### DISCUSSION THREAD

**Discussion by Codecaver** [goharbor/harbor#19836] [Q&A] [answered]
I want to migrate the replication tasks of the old harbor to the new harbor. I know that each task has an id. What should I do to make the replication tasks of the old and new environments consistent. Including the task ID must also be consistent

**Comment by MinerYang** [comment, marked as answer]
we do not  have handing db migration and you need to with the help of db migration tools to do the whole db migrate.
